.stylelintignore

```diff
@@ -0,0 +1 @@
+components/**/*-generated.css
```

Comment:
Could this be done using `components/*/dist/*.css`?

Reply:
I'm not sure. I included this so that we could ignore [these](https://github.com/adobe/spectrum-css/blob/main/components/pickerbutton/pickerbutton-generated.css) [two](https://github.com/adobe/spectrum-css/blob/main/components/infieldbutton/infieldbutton-generated.css) `-generated.css` files that are present in the source (not the `dist`). These were once generated by DSS and then included here. I imagine we'll remove them once these two components get migrated.

Reply:
Oh! I thought this was related to a new asset being created by linting which seemed odd. Yes, that makes sense.